lib/systems/platforms.nix

```diff
@@ -203,6 +203,36 @@ rec {
   # Legacy attribute, for compatibility with existing configs only.
   raspberrypi2 = armv7l-hf-multiplatform;
 
+  zero-gravitas = {
+    name = "zero-gravitas";
+    kernelMajor = "4.1";
+    kernelBaseConfig = "zero-gravitas_defconfig";
+    kernelArch = "arm";
+    # kernelTarget verified by checking /boot on reMarkable 1 device
+    kernelTarget = "zImage";
+    kernelAutoModules = false;
+    kernelDTB = true;
+    gcc = {
+      # The default toolchain environment setup puts -mfloat-abi=hard
+      # in $CC, which is somewhat important---anything that uses the
+      # standard library from this toolchain _must_ be compiled
+      # hardfp. If we put this here, it'll get included in the wrapper
+      # flags, which is basically the same thing.
+      #
+      # However, the Linux kernel doesn't use the standard library and
+      # requires the ability to be built softfp, so we can't just set
+      # =hard. Therefore, we actually mess with add-flags.sh in the
+      # compiler derivation to set -mfloat-abi based on an environment
+      # variable (defaulting to hard), and override that variable in
+      # the kernel.
+      #
+      # It would be nice to make modules also automatically have that variable set, but that appears to not be possible.
+      # float-abi = "hard";
+      fpu = "neon";
```

Comment:
@pl-semiotics Could you clarify how much of an issue this is and what it will affect?